std/os/test.zig

```diff
@@ -149,3 +149,14 @@ test "realpath" {
     var buf: [std.fs.MAX_PATH_BYTES]u8 = undefined;
     testing.expectError(error.FileNotFound, fs.realpath("definitely_bogus_does_not_exist1234", &buf));
 }
+
+test "sigaltstack" {
+    if (builtin.os != .windows and builtin.os != .uefi and builtin.os != .wasi) {
```

Comment:
this is fine. note you can also do this:

`if (builtin.os == .windows or builtin.os == .uefi or builtin.os == .wasi) return error.SkipZigTest;`

also this test isn't run for uefi so you don't need that check.

Reply:
Oh, that looks better.